Update feed inspector UI and metadata display

obenland opened the pull request:
Before | After
--- | ---
<img width="1582" height="1034" alt="Screenshot 2025-11-19 at 11 46 40 AM" src="https://github.com/user-attachments/assets/6186f2ff-db6b-4414-a785-245fa3d0cbd2" /> | <img width="1582" height="1034" alt="Screenshot 2025-11-19 at 12 12 35 PM" src="https://github.com/user-attachments/assets/675c0789-c864-4cdf-a7b5-1632bb2a9386" />


## Summary

This PR improves the feed inspector UI and adds better timestamp handling throughout the feed views.

### Inspector Improvements
- Move close button into card header with X icon instead of text
- Replace date with webfinger in inspector header
- Link author name to profile URL
- Add clickable relative timestamp (5m, 2h, 6d) that links to original post
- Remove Page component wrapper for cleaner, simpler layout
- Make card header border extend full width of panel

### Feed Stage Updates
- Display relative time format (5m, 2h, 6d max) instead of full dates
- Remove link from timestamp in stage (clicking row opens inspector)
- Remove icon from Feed sidebar menu item

### Technical Changes
- Fix content overflow when inspector is open at 50% width
- Add proper word wrapping and width constraints to feed items
- Style Mastodon-formatted links (ellipsis/invisible classes)
- Create shared `getRelativeTimeShort` utility with i18n support
- Use WordPress `dateI18n` for localized date fallback (dates > 6 days)

## Screenshots

[Screenshots would go here showing before/after of inspector UI]

## Testing
- Open feed inspector and verify close button is in header
- Check that author name links to profile
- Verify timestamp shows relative time and links to original post
- Confirm webfinger is displayed
- Test that content doesn't overflow when inspector is at 50% width

obenland committed:
Update feed inspector UI and metadata display

- Move close button into card header with X icon
- Replace inspector date with webfinger, link author name to profile
- Show relative time (5m, 2h, 6d) in feed stage metadata
- Link relative time to original post
- Remove Page component wrapper from inspector for cleaner layout
- Fix content overflow in stage region when inspector is open
- Style links in post content (ellipsis/invisible classes for Mastodon)
- Make card header border extend full width

obenland committed:
Add timestamp linking and remove feed icon

- Add relative time (5m, 2h, 6d) with link to original post in inspector header
- Display webfinger and linked timestamp next to author name in inspector
- Remove time link from feed stage (clicking opens inspector instead)
- Remove icon from Feed menu item in sidebar
- Create shared getRelativeTimeShort utility with i18n support
- Use dateI18n for localized fallback dates older than 6 days

pfefferle commented:
I see a weird date format in both, the stream and the inspector!?

<img width="476" height="170" alt="Screenshot 2025-11-19 at 20 03 37" src="https://github.com/user-attachments/assets/ad768b67-fc10-4664-98e2-39266ee918f9" />

obenland committed:
Use site date format for old posts and replace hardcoded styles with design tokens

- Use site's date format from getSettings() for posts older than a week instead of 'll' format
- Fix week threshold from <= 6 days to < 7 days
- Replace all hardcoded sizes, colors, and spacing in feed styles with wpds design tokens
- Remove redundant diffMinutes < 1 case in getRelativeTimeShort

obenland commented:
> I see a weird date format in both, the stream and the inspector!?

Thanks, looks like Claude thought it could use Moment formatting when `datei18n()` was expecting php-style date formats. Fixed in [b1a3769](https://github.com/Automattic/wordpress-activitypub/pull/2503/commits/b1a37694a1f7ecbb216277758e946d61936a3f39).

pfefferle reviewed (approved):
nice!

pfefferle reviewed (changes requested):
The date on comments is also broken

<img width="1471" height="576" alt="Image" src="https://github.com/user-attachments/assets/4e19c024-46f2-49cb-b048-ca9e00917130" />
